Makefile

```diff
@@ -58,4 +66,8 @@ uninstall:
 
 .PHONY: version
```

Comment:
nit: same as above, maybe just calling this `release`?

Reply:
Discussed offline, but having the two with distinct names makes it harder to brainfart and type the wrong one in. Since anything published to npm is un-revocable, it would be good to prevent accidental publishing of beta code as a release.